src/Core.UnitTests/Analysis/AnalysisIssueTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using SonarLint.VisualStudio.Core.Analysis;
+using SonarLint.VisualStudio.Integration.UnitTests;
+
+namespace SonarLint.VisualStudio.Core.UnitTests.Analysis
+{
+    [TestClass]
+    public class AnalysisIssueTests
+    {
+        [TestMethod]
+        public void IsFileLevel_PrimaryLocationHasTextRange_True()
```

Comment:
The test names are the wrong way round: the "_True" test is checking for a `false` result, and vice versa.
Naming: we'd normally name the tests `xxx_ReturnsTrue/False`